stdlib/mime/src/main/ballerina/src/mime/natives.bal

```diff
@@ -620,7 +620,7 @@ function externHasHeader(Entity entity, string headerName, HeaderPosition positi
 # + contentToBeEncoded - Content that needs to be encoded can be of type `string`, `byte[]` or `io:ReadableByteChannel`
 # + charset - Charset to be used. This is used only with the string input
 # + return - An encoded `string` if the given input is of type string, an encoded `byte[]` if the given input is of
-#            type byte[], an encoded `io:ReadableByteChannel` if the given input is of type `io:ReadableByteChannel`,
+#            type byte[], an encoded `io:ReadableByteChannel` if the given input is of type ``io:ReadableByteChannel``,
```

Comment:
shouldn't this be allowed?
i.e:
```
# some text followed by type `M:X`
```

Spec says "if it is a qualified name M:X, then the source file must have imported M, and X must refer to a public name of an appropriate type in M"

Reply:
this fix was added with fac6140.

seems we have to make an exception for `function` keyword.

Spec says, 
*BFM also recognizes `` `f()` `` as an alternative to `` function `f` ``. **In both cases**, f can have any of the following forms
`x(), m:x(), T.y(), m:T.y()`*

so for `function` keyword, all together 6 formats(`x, m:x, x(), m:x(), T.y(), m:T.y()`) should be allowed right?

Reply:
yea, thats correct.
We can add some parser test - that would be sufficient.